### ISSUE TITLE
App does not seem to pop up on Mac but working on Windows

### ISSUE THREAD

**Issue by flexibo** [nus-cs2103-AY2324S1/forum#191]
## My setup 
Intellij: Community Edition 2023.2.1
Java Version: 11
OS: Windows

## Issue
App does not seem to appear when executing on MAC, but working fine when running on Windows 

### My output when executing the app (Windows)
```
>java -jar cleo.jar
Sep 23, 2023 1:36:41 AM com.sun.javafx.application.PlatformImpl startup
WARNING: Unsupported JavaFX configuration: classes were loaded from 'unnamed module @28900a60'
Sep 23, 2023 1:36:42 AM javafx.fxml.FXMLLoader$ValueElement processValue
WARNING: Loading FXML document with JavaFX API of version 20.0.1 by JavaFX runtime of version 17.0.7 
```
At this point, the app opens on screen. 

### Friend's output when executing the app (Mac)
```
>java -jar cleo.jar
Error loading file, creating an empty list.
Sep 23, 2023 1:34:37 AM javafx.fxml.FXMLLoader$ValueElement processValue
WARNING: Loading FXML document with JavaFX API of version 20.0.1 by JavaFX runtime of version 19.0.4
```
At this point... nothing happens, its just stuck here. 
Note: The "Error loading file...." line is normal, it prints that when it can't find the tasks data file. 

I'm not very sure what went wrong, so I am not sure how to fix it. 

[Here is my latest release so far](https://github.com/flexibo/ip/releases/tag/A-Release)

**Comment by flexibo** [comment]
I'm also not sure if this is relevant information, but before this there was another error, which was that gradle compiled the project in Java 21 instead of Java 11, so the app could not run on my friend's system. I did not change my default java version in my environment until now, so after I did that, rebuild, and make using shadowjar again, that issue was resolved, but it now created this issue. 

Maybe me not changing my environment java version until very late might have something to do with this current issue, I'm not sure.
